usr/lib/python3/dist-packages/linuxmusterLinuxclient7/realm.py

```diff
@@ -123,14 +123,14 @@ def getJoinedDomains():
 
     return True, list(filter(None, result.stdout.split("\n")))
 
-def discoverDomains():
+def discoverDomains(domain=None):
     """
-    Searches for avialable domains on the current network
+    Searches for avialable domains on the current network"realm discover --name-only '" + domain + "'"
```

Comment:
Please revert or change to a normal sentence. Like this, it does not add any valuable information to the comment.